src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -425,21 +419,15 @@ private static void checkKeycodeArgument(int keycode) {
 
     /**
      * Returns the color of a pixel at the given screen coordinates.
-     * <p>
-     * If the desktop environment requires that permissions be granted
-     * to capture screen content, and the required permissions are not granted,
-     * then a {@code SecurityException} may be thrown,
-     * or the content of the returned {@code Color} is undefined.
-     * </p>
```

Comment:
This text should not have been removed. It pertains to the desktop permissions as well as the Java SecurityManager.

Reply:
Ok, I will revert it.

Reply:
The description for the SecurityException thrown by these methods were adjusted to "if access to the screen is denied by desktop environment". If you bring back the paragraphs that were removed then you might have to adjust the wording a bit as otherwise the word "permissions" is ambiguous.

Reply:
Phil, if you have better wording for the `@throws SecurityException` of these methods, let me know; otherwise I will restore the paragraph above and keep the current text for `SecurityException` the same as it is now.

Reply:
I restored the text in https://github.com/openjdk/jdk/pull/21498/commits/adf5ed789b0437fa57c87f29e6a4e6b6f7f0c92b. I've left the `@throws SecurityException` text the same, but let me know if you want that changed.